docker-compose.abuse.yml

```diff
@@ -0,0 +1,34 @@
+version: "3.7"
+
+x-logging: &default-logging
+  driver: json-file
+  options:
+    max-size: "10m"
+    max-file: "3"
+
+services:
+  abuse:
+    build:
+      context: ./docker/abuse
+      dockerfile: Dockerfile
+    container_name: abuse
+    restart: unless-stopped
+    logging: *default-logging
+    env_file:
+      - .env
+    environment:
+      - ABUSE_LOG_LEVEL=${ABUSE_LOG_LEVEL}
+      - ABUSE_MAILBOX=${ABUSE_MAILBOX}
+      - ABUSE_SPONSOR=${ABUSE_SPONSOR}
+      - BLOCKER_API_URL=${BLOCKER_API_URL}
```

Comment:
`malware-scanner` already introduces env vars for this:
```
      - BLOCKER_IP=${BLOCKER_IP:-10.10.10.110}
      - BLOCKER_PORT=${BLOCKER_PORT:-4000}
```
Let's reuse them here.

Reply:
updated